lib/util.js

```diff
@@ -387,13 +387,24 @@ function _fetch_rss_page (feed, page, items, all_items, max_items, oldest_date)
 
 exports.fetch_rss_entries = fetch_rss_entries;
 
+function create_wordpress_paginator (feed) {
+    return function (n) {
```

Comment:
Might be easier to read with `n` expanded to a full var name

Reply:
Fixed